pmm/pmm-server-release.groovy

```diff
@@ -162,9 +165,15 @@ pipeline {
                     sg docker -c "
                         docker pull ${DOCKER_VERSION}
                         docker tag ${DOCKER_VERSION} percona/pmm-server:${VERSION}
+                        docker tag ${DOCKER_VERSION} percona/pmm-server:${DOCKER_MID}
+                        docker tag ${DOCKER_VERSION} percona/pmm-server:${TOP}
```

Comment:
Should it be used variable TOP_VER instead of TOP?

Reply:
fixed. thanks